catalog/anthos-cluster/Kptfile

```diff
@@ -0,0 +1,27 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: anthos-cluster
+  annotations:
+    blueprints.cloud.google.com/title: Anthos cluster blueprint
+upstream:
+  type: git
+  git:
+    repo: https://github.com/GoogleCloudPlatform/blueprints
+    directory: /catalog/gke
```

Comment:
If `anthos-cluster` is a top level blueprint in catalog, I think we should compose GKE under `/catalog/anthos-cluster/gke`. I will defer to @morgante if this is the recommended approach

Reply:
Based on a quick discussion, we want to keep these both as top level.